backend/public-graph/graph/resolver.go

```diff
@@ -1764,22 +1769,40 @@ func (r *Resolver) sendErrorAlert(ctx context.Context, projectID int, sessionObj
 				continue
 			}
 
-			numAlerts := int64(-1)
+			var alertCounts []AlertCountsGroupedByRecent
 			if err := r.DB.Raw(`
-				SELECT COUNT(*)
+				SELECT COUNT(*), ev.sent_at > NOW() - ? * (INTERVAL '1 SECOND') AS recent_alert
```

Comment:
nit: i prefer putting the grouped column first to make this easier to read
```suggestion
				SELECT ev.sent_at > NOW() - ? * (INTERVAL '1 SECOND') AS recent_alert, COUNT(*)
```